setup.cfg

```diff
@@ -6,3 +6,8 @@ max-line-length = 119
     # 'leapp.snactor.fixture.current_actor_libraries' imported but unused
 per-file-ignores =
     repos/system_upgrade/el7toel8/actors/*/tests/*.py:F811,F401
+application-import-names = leapp
+# check only for correct group type and abc sorting without group
+import-order-style = pep8
+# default values + ignore strict alphabetical sorting within one module import
+ignore = E121,E123,E126,E226,E24,E704,W503,W504,I101
```

Comment:
I wonder why are these checks ignored - by default ignored checks also contains E133 and W505 but lacks I101.

Also I would prefer not to use E24, but to explicitly write down E241 adn E242, in case some other E24* check is added to flake8 in the future.

Reply:
Ignore my comment about check I101, I have mistaken it for E101.

Reply:
Hmm the original defaults I took were from flake8 --help
``` 
 --ignore=errors       Comma-separated list of errors and warnings to ignore
                        (or skip). For example, ``--ignore=E4,E51,W234``.
                        (Default: E121,E123,E126,E226,E24,E704,W503,W504)
```
You are saying E133 and W505 are there as well?

Reply:
On the other hand - it's nice having them here.

Reply:
Sorry for yet another confusion, the default configuration that ignores also E133 and W505 is for [pycodestyle](https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes), not for flake8.